js/components/agency_component_preview.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AgencyComponentProcessingTime from './agency_component_processing_time';
+import FoiaPersonnel from './foia_personnel';
+import FoiaSubmissionAddress from './foia_submission_address';
+import PrettyUrl from './pretty_url';
+import { AgencyComponent } from '../models';
+
+
+function AgencyComponentPreview({ agencyComponent }) {
+  const description = AgencyComponent.agencyMission(agencyComponent);
+  const requestUrl = `/request/agency-component/${agencyComponent.id}/`;
+  debugger;
+  return (
+    <div className="agency-preview usa-grid-full">
+      <div className="usa-width-one-whole">
+        <h2>{agencyComponent.agency.name}</h2>
+        <h3>{agencyComponent.title}</h3>
+      </div>
+      <div className="usa-width-one-half">
+        { description &&
+          <div>
+            <h4>Agency mission</h4>
+            <p>{description}</p>
+          </div>
+        }
+
+        { agencyComponent.request_time_stats &&
+          <div>
+            <h4>Median processing time (estimates)</h4>
+            <AgencyComponentProcessingTime
+              requestTimeStats={agencyComponent.request_time_stats}
+            />
+          </div>
+        }
+
+        <h4>
+          The records or information you&apos;re looking for may already be public.
+        </h4>
+        <p>You can find out by reaching out to the agency or by visiting
+        their <a href={agencyComponent.website.uri}>website</a> or
+        FOIA <a href={agencyComponent.reading_rooms[0].uri}>reading room</a>.</p>
```

Comment:
I don't think we can guarantee that every agency component has this data. I think we need to check for `readings_rooms` and `website`. In the AgencyComponent model I think they default to `[]` and `{}`, but we still need to check the `uri` exists or not.

Reply:
Cool, I'll implement that as a conditional similar to `request_time_stats` unless there's a different way I should do it!